pod/video/templates/videos/video_row_select.html

```diff
@@ -0,0 +1,129 @@
+                {% if not in_favorites_playlist and can_edit_or_remove %}
+                    <li><a href="{% url 'playlist:remove-video' playlist.slug video.slug %}"
+                           title="{% trans 'Remove from playlist' %}" class="dropdown-item"
+                           id="remove-from-playlist-btn-{{ video.id }}">{% trans "Remove from playlist" %}</a></li>
+                {% endif %}
+            {% endif %}
+            {{ is_favorite }}
+            {% is_favorite user video as is_favorite %}
+            {% get_favorite_playlist user as fav_playlist %}
+            {% if USE_PLAYLIST and USE_FAVORITES and video.is_draft is False and not hide_favorite_link is True %}
+                <li>
+                    {% if is_favorite %}
+                        <a href="{% url 'playlist:remove-video' fav_playlist.slug video.slug %}"
+                           title="{% trans 'Remove from favorite' %}" class="dropdown-item"
+                           id="favorite-btn-{{ video.id }}" aria-pressed="true"
+                           role="button">{% trans "Remove from favorite" %}</a>
+                    {% else %}
+                        <a href="{% url 'playlist:add-video' fav_playlist.slug video.slug %}"
+                           title="{% trans 'Add in favorite' %}" class="dropdown-item" id="favorite-btn-{{ video.id }}"
+                           aria-pressed="false" role="button">{% trans "Add in favorite" %}</a>
+                    {% endif %}
+                </li>
+            {% endif %}
+            {% if video.owner == request.user or request.user.is_superuser or request.user in video.additional_owners.all or perms.video.change_video %}
+                <li>
+                    <a href="{% url 'video:video_edit' slug=video.slug %}" title="{% trans 'Edit the video' %}"
+                       class="dropdown-item">{% trans 'Edit the video' %}</a>
+                </li>
+            {% endif %}
+            {% if video.owner == request.user or request.user.is_superuser or request.user in video.additional_owners.all or perms.completion.add_contributor or perms.completion.add_track or perms.completion.add_document or perms.completion.add_overlay %}
+                <li>
+                    <a href="{% url 'video:completion:video_completion' slug=video.slug %}"
+                       title="{% trans 'Complete the video' %}"
+                       class="dropdown-item">{% trans 'Complete the video' %}</a>
+                </li>
+            {% endif %}
+            {% if video.encoded and video.encoding_in_progress is False %}
+                {% if video.owner == request.user or request.user.is_superuser or perms.cut.cut_video %}
+                    <li>
+                        <a href="{% url 'video:cut:video_cut' slug=video.slug %}" title="{% trans 'Cut the video' %}"
+                           aria-hidden="true" class="dropdown-item">{% trans 'Cut the video' %}</a>
```

Comment:
Ne pas mettre un title s'il est rigoureusement identique au contenu du lien.

Reply:
Il vaut mieux le supprimer du coup ? ça remet en cause toute mes croyances sur l'accessibilité O.o

Reply:
dans ce cas, le supprimer oui. Ou faire en sorte qu'il donne une info supplémentaire du genre "'Cut the video <nom de la vidéo>'

Reply:
Aaaah mon dieu, je n'avais pas vu cet affreux "aria-hidden='true'" sur ce lien. pourquoi cacher ce lien aux mal voyants ?

Reply:
Retirez ce hidden que je ne saurais voir...